apps/ghostfolio/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3.9"
+
+services:
+  ghostfolio:
+    container_name: ghostfolio
+    image: ghostfolio/ghostfolio:1.300.0
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:3333
+    environment:
+      NODE_ENV: production
+      HOST: 0.0.0.0
+      PORT: 3333
+      ACCESS_TOKEN_SALT: $GHOSTFOLIO_ACCESS_TOKEN_SALT
+      DATABASE_URL: postgresql://ghostfolio:${GHOSTFOLIO_DB_PASSWORD}@ghostfolio-postgres:5432/ghostfolio?sslmode=prefer
+      JWT_SECRET_KEY: ${GHOSTFOLIO_JWT_SECRET_KEY}
+      POSTGRES_DB: ghostfolio
+      POSTGRES_USER: ghostfolio
+      POSTGRES_PASSWORD: ${GHOSTFOLIO_DB_PASSWORD}
+      REDIS_HOST: ghostfolio-redis
+      REDIS_PASSWORD: ${GHOSTFOLIO_REDIS_PASSWORD}
+      REDIS_PORT: 6379
+    networks:
+      - tipi_main_network
+    depends_on:
+      - ghostfolio-postgres
+      - ghostfolio-redis
+    labels:
+      # Main
+      traefik.enable: true
+      traefik.http.middlewares.ghostfolio-web-redirect.redirectscheme.scheme: https
+      traefik.http.services.ghostfolio.loadbalancer.server.port: 3333
+      # Web
+      traefik.http.routers.ghostfolio-insecure.rule: Host(`${APP_DOMAIN}`)
+      traefik.http.routers.ghostfolio-insecure.entrypoints: web
+      traefik.http.routers.ghostfolio-insecure.service: ghostfolio
+      traefik.http.routers.ghostfolio-insecure.middlewares: ghostfolio-web-redirect
+      # Websecure
+      traefik.http.routers.ghostfolio.rule: Host(`${APP_DOMAIN}`)
+      traefik.http.routers.ghostfolio.entrypoints: websecure
+      traefik.http.routers.ghostfolio.service: ghostfolio
+      traefik.http.routers.ghostfolio.tls.certresolver: myresolver
+      # Local domain
+      traefik.http.routers.ghostfolio-local-insecure.rule: Host(`ghostfolio.${LOCAL_DOMAIN}`)
+      traefik.http.routers.ghostfolio-local-insecure.entrypoints: web
+      traefik.http.routers.ghostfolio-local-insecure.service: ghostfolio
+      traefik.http.routers.ghostfolio-local-insecure.middlewares: ghostfolio-web-redirect
+      # Local domain secure
+      traefik.http.routers.ghostfolio-local.rule: Host(`ghostfolio.${LOCAL_DOMAIN}`)
+      traefik.http.routers.ghostfolio-local.entrypoints: websecure
+      traefik.http.routers.ghostfolio-local.service: ghostfolio
+      traefik.http.routers.ghostfolio-local.tls: true
+
+  ghostfolio-postgres:
+    container_name: ghostfolio-postgres
+    image: postgres:14
```

Comment:
if it can use v15 of postgres, prefer it. As it will be complicated later for upgrade once this app is deployed among users.
Also, alpine version is preferred: `docker.io/library/postgres:15.4-alpine`